client/src/pages/Home.tsx

```diff
@@ -202,9 +216,14 @@ function Home(this: any) {
             </div>
           </div>
           <button
+            id="summarise-button-id"
             onClick={getSummary}
-            style={{ backgroundColor: "#2e7faa" }}
+            style={{
+              backgroundColor: "#2e7faa",
+              cursor: isButtonDisabled ? "not-allowed" : "pointer",
+            }}
             className="mt-8 ml-52 py-2 px-4 text-white rounded"
+            disabled={isButtonDisabled}
```

Comment:
It's good that we have the _disabled_ button for the less than 100 words case. However, it would be good to give feedback to the User that they need _at least_ 100 words.